src/intents.js

```diff
@@ -3,7 +3,7 @@ import {cozyFetchJSON} from './fetch'
 const intentClass = 'coz-intent'
 
 // inject iframe for service in given element
-function injectService (url, element, data) {
+function injectService (url, element, intent, data) {
   const document = element.ownerDocument
   if (!document) throw new Error('Cannot retrieve document object from given element')
```

Comment:
Yes it could be I think the reason is that we should do
```js
const document = element.ownerDocument
```
twice, in two different part of the code. We may think about it in a next dedicated PR.